packages/components/src/select-control/styles/select-control-styles.ts

```diff
@@ -138,17 +146,14 @@ export const Select = styled.select< SelectProps >`
 `;
 
 export const DownArrowWrapper = styled.div`
-	align-items: center;
-	bottom: 0;
-	display: flex;
-	padding: 0 4px;
-	pointer-events: none;
-	position: absolute;
-	top: 0;
-
-	${ rtl( { right: 0 } ) }
+	margin-inline-end: ${ space( -1 ) }; // optically adjust the icon
+	line-height: 0;
+`;
 
-	svg {
-		display: block;
-	}
+export const InputControlSuffixWrapperWithClickThrough = styled(
+	InputControlSuffixWrapper
+)`
+	position: absolute;
+	right: 0;
```

Comment:
I believe these styles are missing RTL support:

```suggestion
	${ rtl( { right: 0 } ) }
```